src/NzbDrone.Core/Update/InstallUpdateService.cs

```diff
@@ -170,7 +182,12 @@ public void Execute(ApplicationUpdateCommand message)
 
         public void Execute(InstallUpdateCommand message)
         {
-            InstallUpdate(message.UpdatePackage);
+            var success = InstallUpdate(message.UpdatePackage);
+
+            if (!success)
+            {
+                throw new NzbDroneClientException(System.Net.HttpStatusCode.Conflict, "Failed to install update");
```

Comment:
Right now we do it through command progress messaging, I'm not sure that this exception will even make it out of the command executor.